services/userService.js

```diff
@@ -15,6 +15,36 @@ const signUp = async( userName, password, email, phoneNumber, address, birth, ge
   return userDao.createUser(userName, hashedPassword, email, phoneNumber, address, birth, gender, point)
 }
 
+
+const signIn = async( email, password ) => { 
+  const user = await userDao.getUserByEmail(email);
+
+  if(!user){
+    const error = new Error('WRONG_EMAIL')
+    error.statusCode = 401;
+
+    throw error;
+  }
+  const match = await bcrypt.compare(password, user.password);
+
+  if(!match){
+    const error = new Error('WRONG_PASSWORD')
```

Comment:
계정이 잘못 되었는지, 비밀번호가 잘못 되었는지 모르게 메시지를 주는 것이 더 좋습니다.
```suggestion
    const error = new Error('INVALID_USER')
```